app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt

```diff
@@ -255,6 +260,14 @@ class PatientEntryScreen(context: Context, attrs: AttributeSet) : RelativeLayout
     consentTextView.text = consentText
   }
 
+  private fun setConsentLabelText() {
+    val consentLabelTextResId: Int = when (country.isoCountryCode) {
+      Country.BANGLADESH -> R.string.patiententry_consent_sms_reminders
+      else -> R.string.patiententry_consent_whatsapp_sms_reminders
+    }
+    consentLabel.setText(consentLabelTextResId)
```

Comment:
Can we refactor this to remove the type `Int` and change `setText` to `text` in order to make it more Kotlin idiomatic?
```
    val consentLabelTextResId = when (country.isoCountryCode) {
      Country.BANGLADESH -> R.string.patiententry_consent_sms_reminders
      else -> R.string.patiententry_consent_whatsapp_sms_reminders
    }
    consentLabel.text = resources.getString(consentLabelTextResId)
```

Reply:
This doesn't really change much. I prefer to keep it as it is.

Reply:
It does not make sense to have the type specified as it is derived already.
Can you please make that change atleast?